makefile.shade

```diff
@@ -769,7 +769,7 @@ var CI_DARWIN_DROP_PATH = '${Environment.GetEnvironmentVariable("CI_DARWIN_DROP_
                         {
                             if(framework == target.Framework)
                             {
-                                KTest(projectFile);
+                                DnxTest(projectFile, false);
```

Comment:
This is `false` because it's useless for `DnuTest` to do a single project in parallel. (The parameter is `testParallel`.)

I'll check again on named arguments. Think that failed last I tried though I see a few named parameters elsewhere.